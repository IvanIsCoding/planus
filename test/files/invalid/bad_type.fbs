table Foo {
  x0: bool = 1;
  x1: uint8 = true;
  x2: uint8 = NaN;
  x3: string = 42;
  x4: uint8 = "string";
}