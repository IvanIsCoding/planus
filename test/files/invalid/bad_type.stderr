[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_type.fbs:2:14
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x0: bool = [0m[31m1[0m;
  [0m[34mâ”‚[0m              [0m[31m^[0m [0m[31mExpression not of the correct type[0m

[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_type.fbs:3:15
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   x1: uint8 = [0m[31mtrue[0m;
  [0m[34mâ”‚[0m               [0m[31m^^^^[0m [0m[31mExpression not of the correct type[0m

[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_type.fbs:4:15
  [0m[34mâ”‚[0m
[0m[34m4[0m [0m[34mâ”‚[0m   x2: uint8 = [0m[31mNaN[0m;
  [0m[34mâ”‚[0m               [0m[31m^^^[0m [0m[31mExpression not of the correct type[0m

[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_type.fbs:5:16
  [0m[34mâ”‚[0m
[0m[34m5[0m [0m[34mâ”‚[0m   x3: string = [0m[31m42[0m;
  [0m[34mâ”‚[0m                [0m[31m^^[0m [0m[31mExpression not of the correct type[0m

[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_type.fbs:6:15
  [0m[34mâ”‚[0m
[0m[34m6[0m [0m[34mâ”‚[0m   x4: uint8 = [0m[31m"string"[0m;
  [0m[34mâ”‚[0m               [0m[31m^^^^^^^^[0m [0m[31mExpression not of the correct type[0m

Error: Bailing because of errors
